backend/supabase/seed.sql

```diff
@@ -0,0 +1,36 @@
+INSERT INTO auth.users(instance_id, id, aud, "role", email, encrypted_password, email_confirmed_at, raw_app_meta_data, raw_user_meta_data, is_sso_user)
+  VALUES ('00000000-0000-0000-0000-000000000000', gen_random_uuid(), 'authenticated', 'authenticated', 'hitasp@outlook.com', app.generate_token(), now(), '{"provider": "email", "providers": ["email"], "account_name": "hitasp"}', '{"display_name": "Hitasp", "bio": "A Test Account"}', FALSE);
```

Comment:
fyi some test data are in here. This is not a concern rn since this won't be use in production but your info is here.  Should probably be removed before merging in.

Reply:
yes, it's for testing triggers. I forgot to delete.